### PR TITLE
Update header for native crypto / ERC 20 token screens #2007

### LINKED ISSUE #2007: Update header for native crypto / ERC 20 token screens
This is related to #1995. The part that should change is circled in red:

https://app.zeplin.io/project/5d0356be2f25b319dcc5944b/styleguide/components?cseid=5efeeb8d3e258fb78e39e09b

Refer to https://github.com/AlphaWallet/alpha-wallet-ios/issues/1995#issuecomment-653405327 to see how the line "Market Price: xxx" changes when tapped. There are 3 states. Start with "Value: xxx", then "Market Price: xxx", then "Available: ", then circle back again. Ignore "Earned so far: xxx" in Zeplin.

Notice that like the cells in Wallet tab, there are at least a few variations:

* native crypto where fiat value is available
* native crypto where fiat value is not available
* tokens, no fiat value
* native crypto or tokens for testnets

Can you see if there are other screens that uses this display or a very similar display? We might want to update those too.

### PR DESCRIPTION
**Pull request by vladyslav-iosdev**
Closes #2007

### CODE REVIEW

File: AlphaWallet/Transfer/Views/SendHeaderView.swift [hunk 1]
```diff
@@ -116,49 +220,30 @@ class SendHeaderView: UIView {
...
 
-        if viewModel.showAlternativeAmount {
-            viewsToShowOnlyIfAlternativeAmountsAreAvailable.showAll()
-        } else {
-            viewsToShowOnlyIfAlternativeAmountsAreAvailable.hideAll()
-        }
-    }
-
-    @objc private func showContractWebPage() {
-        delegate?.didPressViewContractWebPage(inHeaderView: self)
```
Comment (hboon) [inline AlphaWallet/Transfer/Views/SendHeaderView.swift:162]: Sorry, I forgot to mention this. Can you make tapping the token icon open browser at the Etherscan (or blockscout, etc) page showing this token? I think this is the line of code that does that. Before the PR you can tap on the words "Ethereum Blockchain" to trigger this.

**Comment by vladyslav-iosdev** [comment]
> Looks good. I understand it's a WIP. There shouldn't be "Market Price" for ERC20s (which don't have a price/ticker):
> 
> 
> 
> Also tap to cycle between "Market Price" etc when it's available.

yep, its wip.

**Comment by hboon** [comment]
@vladyslav-iosdev is this good for review?

**Comment by vladyslav-iosdev** [comment]
> @vladyslav-iosdev is this good for review?

yep

**Comment by vladyslav-iosdev** [comment]
> * [ ]  Discover token (0x8C0Edb69eBF038BA0c7a4873E40fC09725064c2e on mainnet) shouldn't show `Value: xxx`
> *

this one detected as ERC20Token
here as i undarstand right, old map for displaying states
native crypto where fiat value is available         -   marketPriceLabel
native crypto where fiat value is not available   -   earnedSoFarLabel
tokens, no fiat value                                             -   valueLabel
native crypto or tokens for testnets                   -   blockChainTagLabel

and now it should be like this, right?

fiat value is available                                            -   marketPriceLabel
tokens, no fiat value                                             -   valueLabel
native crypto or tokens for testnets                    -   blockChainTagLabel

**Comment by vladyslav-iosdev** [comment]
> Oh, I get where we misunderstood each other now :)
> 
> * [ ]  Here's how it should be:
> 
> if native crypto where fiat value (i.e price/ticker) is available (so just ETH for mainnet and xDai):
> 
> 1. show value
> 2. tap on value and switch to show Market Price
> 3. tap on the market price to show value again, i.e. looping back to (1)
> 
> (Ignore "Available", "Earned so Far". I said to include "Available" previously, but just realised we don't have a value for that yet)
> 
> If fiat value is not available:
> 
> A. If it's a testnet (`RPCServer.isTestnet`), show the blockchain label
> B. If it's not a testnet, hide the row completely. So this will be basically be all ERC20 tokens on non-testnets
> 
> * [ ]  Showing the token name and symbol at the navigation bar is good. Can you remove the name below the token icon? It seems to be showing up for ERC20 (but not native cryto). Hide it for both cases, only show the symbol. So instead of "1 AlphaWallet Discover (ALP)", only show "1 ALP", and for native crypto too: instead of "0.0997 (ETH)", just show "0.90997 ETH"

now should be ok.

**Comment by hboon** [comment]
Forced pushed after rebase master

**Pull request merged**